Jenkinsfile_nightly

```diff
@@ -24,4 +24,5 @@ withNightlyPipeline(type, product, component) {
     env.Rules = params.SecurityRules
     enableSlackNotifications(channel)
     enableSecurityScan()
```

Comment:
Swapping temporarily the order of functional tests and security scan will enable us to run the functional tests first

Reply:
does the order matter ?

Reply:
We could run the functional tests first if security is failing